packages/miniflare/src/index.ts

```diff
@@ -2420,6 +2420,14 @@ export class Miniflare {
 			cfBlob
 		);
 
+		if (process.platform === "win32") {
+			// On Windows, keep-alive connections can sometimes cause issues with sockets
+			// disconnecting unexpectedly. To mitigate this, try to avoid keep-alive race
+			// conditions by telling the runtime to close the connection immediately after
+			// the request is complete
+			forward.headers.set("Connection", "close");
+		}
+
```

Comment:
This seems to be unrelated. Is it meant to be in this PR?

Reply:
Oops!